nixos/modules/system/boot/systemd-unit-options.nix

```diff
@@ -210,6 +210,16 @@ in rec {
       '';
     };
 
+    startLimitIntervalSec = mkOption {
+       default = 0;
```

Comment:
Probably shouldn't have a default.

Reply:
I'm not sure I understand. When default is set as it is here, the attribute is not written out to the systemd unit file. Why it isn't written isn't quite clear to me.

If I remove the `default = 0;` line, I get `The option `systemd.services.audit.startLimitIntervalSec' is used but not defined.` for all services.